src/gui/MainWindow.cpp

```diff
@@ -193,7 +193,9 @@ void MainWindow::toggleMenuBar(MainWindow* win) {
 void MainWindow::initXournalWidget() {
     GtkWidget* boxContents = get("boxContents");
 
-    if (control->getSettings()->isTouchWorkaround()) {
+    // TODO: Awaiting review
+    if (false) {
+//    if (control->getSettings()->isTouchWorkaround()) {
```

Comment:
This is just a quick hack to force the updated scroll handling. I will remove this block entirely if we decide to deprecate the touch workaround.

Reply:
Doing it in followup PR.